notebooks/transport/http.go

```diff
@@ -1,29 +1,26 @@
 package transport
```

Comment:
The logic in the handlers was pretty much completely revamped, so looking at the diff for this file might not be very helpful!